### DISCUSSION TITLE
unenroll exploit

### DISCUSSION THREAD

**Discussion by stuartcat** [3kh0/ext-remover#1178] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
this is just a guide for people who arent in the titanium network discord (.gg/unblock) credits to them
Powerwash (ctrl + alt + shift + r on the login screen) (IF THIS IS BLOCKED FOR YOU, FOLLOW THESE STEPS TO FORCE A POWERWASH: esc + refresh + power, ctrl+d, enter, space, enter)
Proceed through setup as normal
When it starts to enroll wait 1 to 2 seconds (no longer) and perform an EC reset (refresh + power)
When it turns on, if it says "getting your device ready" immediately, it should work! But if it opens to the Welcome page, it didn't work, and you have to restart from step 1.
When it starts to enroll again wait 1 to 2 seconds (no longer) and enter recovery (esc + refresh + power)
Press the power button to turn it off, and leave if off for at least 15 hours. ANYTHING UNDER 15 HOURS WILL NOT WORK but anything above 15 hours will work
When you turn it on you should be unenrolled!

again this isn't my exploit its just a copy paste form discord for people that cant use discord.

**Comment by C0D3Hermit** [comment]
Does this work for v129 and up?

**Reply by SpaceMonke135** [reply]
idk
